### PR TITLE
add pageView to dashboard configuration

### PR DESCRIPTION
**Pull request by noanu**

### CODE REVIEW

File: test/models.spec.ts [hunk 1]
```diff
@@ -270,6 +271,40 @@ describe('Unit | Models', function () {
...
+          accessToken: 'fakeAccessToken',
+          pageView: 'pageView',
+        }
+      };
+
+      // Act
+      const errors = models.validateDashboardLoad(testData.load);
+
+      // Assert
+      expect(errors).toBeUndefined();
```
Comment (ranbreuer) [inline test/models.spec.ts]: can this be undefined if pageView is wrong?

File: test/models.spec.ts [hunk 2]
```diff
@@ -270,6 +271,40 @@ describe('Unit | Models', function () {
       // Assert
       expect(errors).toBeUndefined();
     });
+
+    it(`should return undefined if id and accessToken and pageView are provided`, function () {
```
Comment (ArinaHantsis) [inline test/models.spec.ts]: Shouldn't according to semantic

Comment (noanu) [inline test/models.spec.ts]: why? errors is expected to be undefined.

File: test/models.spec.ts (hunk 1 above)
Comment (noanu) [inline test/models.spec.ts]: no - it only check is it is a string

**Pull request merged**